Neos.Flow/Classes/Cache/AnnotationsCacheFlusher.php

```diff
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+namespace Neos\Flow\Cache;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Configuration\ConfigurationManager;
+use Neos\Flow\Reflection\ReflectionService;
+use Neos\Flow\Log\Utility\LogEnvironment;
+use Psr\Log\LoggerInterface;
+
+final class AnnotationsCacheFlusher
```

Comment:
This should be a singleton, right?

Reply:
Yep. Though only necessary if you want to work with the insance state (see your PR). Right now the class is stateless.